### PR TITLE
Edit organization name and description

### PR DESCRIPTION
**Pull request by dannysullivan**

### CODE REVIEW

**Comment by dannysullivan** [comment]
@jgoley this should be all set for you to style

**Comment by jgoley** [comment]
Cool, I'll take a look.

**Comment by jgoley** [comment]
I updated the functionality of org name editing like we discussed: updates on form submit (so the user can hit enter to save).
All test seem to be passing locally on my end but maybe the need to be updated to reflect the change in functionality?

Let me know if the buttons are weird...A part of me feels like they should be more pronounced and separated from the form elements.

**Comment by dannysullivan** [comment]
@aslagle @nathanathan this is ready to be merged once it's been reviewed

File: packages/controllers/organization_detail.coffee [hunk 1]
```diff
@@ -67,3 +103,15 @@ if Meteor.isServer
       organization = Organizations.findOne(orgId)
       if organization.userIsAdmin(@userId)
         organization.removeAdmin(profileId)
+
+    updateName: (name, id) ->
```
Comment (aslagle) [inline packages/controllers/organization_detail.coffee]: Meteor has a single namespace for methods so these might need to be more specific.

File: packages/views/organization_detail.jade [hunk 2]
```diff
@@ -1,23 +1,48 @@
...
-              span.member-count (#{memberCount})
-            table.table.member-table
+    section
+      .container
+        if editingName
+          form.edit-name
+            input#organization-name.form-control(type="string" name="name" value=organization.name)
+            .actions.middle
+              button.btn-plain.save.save-name-button(type="submit")
+                i.fa.fa-check-circle
```
Comment (aslagle) [inline packages/views/organization_detail.jade]: I thought the check looked like it was validating the input - not sure it's totally clear you should click it to save.

File: packages/controllers/organization_detail.coffee [hunk 3]
```diff
@@ -67,3 +103,15 @@ if Meteor.isServer
...
+      organization = Organizations.findOne({_id: id, createdById: this.userId})
+      organization.set({name: name})
+      organization.save ->
+        organization
```
Comment (nathanathan) [inline packages/controllers/organization_detail.coffee]: Is there a reason you're using the save callback here?

File: packages/views/organization_detail.jade [hunk 4]
```diff
@@ -1,23 +1,48 @@
...
+          .panel.panel-default
+            .panel-heading Members
+            .panel-body
+              unless userIsMember
+                a.btn.btn-default.join-organization Join
+            table.table
               thead
                 tr
                   th Name
+                  th Email
```
Comment (nathanathan) [inline packages/views/organization_detail.jade]: I thought we decided to remove this column due to the complications of only publishing emails that users consented to make public.

**Comment by aslagle** [comment]
This branch needs master merged in and has some outstanding comments.

File: packages/views/organization_detail.jade (hunk 2 above)
Comment (jgoley) [inline packages/views/organization_detail.jade]: I was worried about that. I'll change them into buttons.
I suppose, in the future, if there are more fields to edit, we could redirect to an edit page.

**Comment by jgoley** [comment]
I've updated the editing buttons.
@dannysullivan I think the other comments may apply to your commits? Let me if I can help.

**Comment by dannysullivan** [comment]
@jgoley you're correct, I'll take care of this as soon as I'm done with my current Tater task.

**Comment by dannysullivan** [comment]
@aslagle this is ready

**Comment by aslagle** [comment]
It looks like the organization description doesn't retain single new lines - two new lines works, and shows as two new lines, but one new line gets removed. I also noticed that if you make a very long organization name, it overlaps the description in the org list. Both of those might be unrelated to this.

**Comment by aslagle** [comment]
I'm also getting some errors in the console, but it might be from a very old user account:

Exception in template helper: TypeError: Cannot read property 'userIsAdmin' of undefined
    at Object.Template.organizationDetail.helpers.userIsAdmin

**Pull request closed without merging**